cmd/operator-sdk/bundle/create.go

```diff
@@ -268,3 +273,25 @@ func isPackageManifestsDir(dir, operatorName string) bool {
 	_, err := semver.ParseTolerant(filepath.Clean(filepath.Base(dir)))
 	return isExist(packageManifestPath) && err == nil
 }
+
+// copyScorecardConfigToBundle checks if bundle.Dockerfile and scorecard config exists in
+// the operator project. If it does, it injects the scorecard configuration into bundle
+// image.
+func copyScorecardConfigToBundle() {
+	dockerfile := dockerFilePath()
+	if isNotExist(dockerfile) || isNotExist(scorecardConfigPath) {
+		return
+	}
+
+	var scorecardFileContent string
+
+	scorecardFileContent += fmt.Sprintf("COPY %s %s\n", scorecardConfigPath, "tests/scorecard/")
```

Comment:
It would be cleaner/less error-prone to use a dockerfile [parsing library](https://github.com/moby/buildkit/blob/master/frontend/dockerfile/parser/parser.go#L236).